pkgs/development/python-modules/sentry-sdk/1.nix

```diff
@@ -116,7 +116,7 @@ buildPythonPackage rec {
       # upgrade dependencies, so don't bother testing them.
       "tests/integrations/"
     ]
-    ++ lib.optionals (stdenv.buildPlatform != "x86_64-linux") [
+    ++ lib.optionals (stdenv.buildPlatform.system != "x86_64-linux") [
       # test crashes on aarch64
       "tests/test_transport.py"
```

Comment:
Based on the comment, I'm guessing this should be `!buildPlatform.isAarch64`

Reply:
cc authors: @gador and @SuperSandro2000

Reply:
To be fair, I only tested it on `aarch64-linux` and `x86_64-linux`. I don't know wheter it would fail for other systems (e.g. `x86_64-darwin`). OTOH we can just try to go with `buildPlatform.isAarch64` and see how it works

Reply:
I've changed it to use `buildPlatform.isAarch64` in my last rebase.